frontend/src/scenes/saved-insights/SavedInsights.tsx

```diff
@@ -0,0 +1,151 @@
+import { Button, Col, Row, Table, Tabs } from 'antd'
+import { ColumnType } from 'antd/lib/table'
+import { useActions, useValues } from 'kea'
+import { Link } from 'lib/components/Link'
+import { ObjectTags } from 'lib/components/ObjectTags'
+import { createdByColumn } from 'lib/components/Table/Table'
+import { humanFriendlyDetailedTime } from 'lib/utils'
+import React, { useEffect, useState } from 'react'
+import { userLogic } from 'scenes/userLogic'
+import { DashboardItemType } from '~/types'
+import { savedInsightsLogic } from './savedInsightsLogic'
+import {
+    StarOutlined,
+    StarFilled,
+    LeftOutlined,
+    RightOutlined,
+    UnorderedListOutlined,
+    AppstoreFilled,
+} from '@ant-design/icons'
+import './SavedInsights.scss'
+const { TabPane } = Tabs
+
+export function SavedInsights(): JSX.Element {
+    const { loadInsights, updateFavoritedInsight, loadPaginatedInsights } = useActions(savedInsightsLogic)
+    const { insights, count, offset, nextResult, previousResult, insightsLoading } = useValues(savedInsightsLogic)
+    const { hasDashboardCollaboration } = useValues(userLogic)
+    const [displayedColumns, setDisplayedColumns] = useState([] as ColumnType<DashboardItemType>[])
+
+    useEffect(() => {
+        loadInsights()
+    }, [])
+
+    useEffect(() => {
+        if (!hasDashboardCollaboration) {
+            setDisplayedColumns(columns.filter((col) => !col.dataIndex || !['tags'].includes(col.dataIndex.toString())))
+        } else {
+            setDisplayedColumns(columns)
+        }
+    }, [hasDashboardCollaboration])
+
+    const columns = [
+        {
+            title: 'Name',
+            dataIndex: 'name',
+            key: 'name',
+            render: function renderName(
+                name: string,
+                {
+                    short_id,
+                    id,
+                    description,
+                    favorited,
+                }: { short_id: string; id: number; description?: string; favorited?: boolean }
+            ) {
+                return (
+                    <Col>
+                        <Row>
+                            <Link to={`/i/${short_id}`} style={{ marginRight: 12 }}>
```

Comment:
Problem with linking with the short ID is that the back button won't work if you want to get back to this view